### PR TITLE
update workflow serialized response

### LINKED ISSUE #1520: Expose workflows' public_gold_standard field
I can put a value into `public_gold_standard` just fine, and it works as expected (thanks @camallen) but it doesn't seem to be included in the workflow serialization.

### PR DESCRIPTION
**Pull request by camallen**
closes #1520 added public_gold_standard and added default for null finished_at

### CODE REVIEW

File: app/serializers/workflow_serializer.rb [hunk 1]
```diff
@@ -49,4 +49,8 @@ def content
   def retirement
     @model.retirement_with_defaults
   end
+
+  def finished_at
+    @model.finished_at || ""
```
Comment (marten) [inline app/serializers/workflow_serializer.rb]: Why is an empty string cleaner than a null? Is that just because the parser in JS-land is stupid?

Comment (camallen) [inline app/serializers/workflow_serializer.rb]: something like that. we agreed a while back to always serialize a default value (mostly via db defaults) so i just added it in for consistent behaviour.

Comment (brian-c) [inline app/serializers/workflow_serializer.rb]: I'd actually expect this to be null, since it's an unset date, not text.

Comment (camallen) [inline app/serializers/workflow_serializer.rb]: cool, it'll serialize as a string by default. will remove the empty string.

**Pull request merged**